### DISCUSSION TITLE
Currency Change

### DISCUSSION THREAD

**Discussion by petrk94** [kimai/kimai#3522] [Questions & Answers]
Hello,
Im a new user with Kimai, I tried to change the € Euro currency to Dinar, this works, but the currency character "€" remains, what might cause misinterpreting of the hour rate. Is there a way to solve this or is it a bug? On the Super Admin settings, everything is set correctly.

Hope someone can help :)
Thanks

**Comment by kevinpapst** [comment]
There are several currency settings.
The default currency for new customers and then each customer has its own currency.

> but the currency character "€" remains

Where? Screenshot or explanation please
